### PR TITLE
Added some more docs

### PR DESCRIPTION
**Pull request by Maheliusz**

### CODE REVIEW

File: mrc/localization/color/utils/circle_points.py [hunk 1]
```diff
@@ -2,7 +2,20 @@
 
 
 class CirclePoints:
+    """
+    Class responsible for helping to iterate through circle points.
```
Comment (Lukasz1928) [inline mrc/localization/color/utils/circle_points.py]: Class responsible for getting all points inside a circle? Or sth similar, I think.

File: mrc/localization/offset_probability/probability.py [hunk 2]
```diff
@@ -4,15 +4,56 @@
...
+    Class responsible for calculation probability of robot being in given angle
+
+    Intended for internal usage.
+    """
     def __init__(self, sigma, mu, coef):
+        """
+        Parameters
+        ----------
+        sigma : float
+            Gaussian function parameter
```
Comment (Lukasz1928) [inline mrc/localization/offset_probability/probability.py]: sigma = standard deviation

File: mrc/localization/offset_probability/probability.py [hunk 3]
```diff
@@ -4,15 +4,56 @@
...
+        mu : float
+            Gaussian function parameter
```
Comment (Lukasz1928) [inline mrc/localization/offset_probability/probability.py]: mu = expected value / mean

File: mrc/localization/position/probability.py [hunk 4]
```diff
@@ -4,21 +4,65 @@
...
         d = math.pi - self.mean
         p1 = gauss_distribution(rotation, self.mean, self.stdev)
         p2 = gauss_distribution(rotation, -math.pi - d, self.stdev)
         return max(p1, p2)
 
     def reset(self):
+        """
+        Reset coefficient to original value
+        """
         self.stdev /= self.coef
```
Comment (Lukasz1928) [inline mrc/localization/position/probability.py]: I'm not sure if that's correct but lets let it be for now

File: mrc/localization/color/utils/circle_points.py (hunk 1 above)
Comment (Maheliusz) [reply mrc/localization/color/utils/circle_points.py]: Done

File: mrc/localization/offset_probability/probability.py (hunk 2 above)
Comment (Maheliusz) [reply mrc/localization/offset_probability/probability.py]: Done

File: mrc/localization/offset_probability/probability.py (hunk 3 above)
Comment (Maheliusz) [reply mrc/localization/offset_probability/probability.py]: Done

File: mrc/localization/position/probability.py (hunk 4 above)
Comment (Maheliusz) [reply mrc/localization/position/probability.py]: Okay

**Pull request merged**